examples/helm-on-kubernetes-example/src/main/resources/application.properties

```diff
@@ -0,0 +1,22 @@
+dekorate.helm.name=myChart
+# Normal use case
+dekorate.helm.values[0].property=helmOnKubernetesExample.name
+dekorate.helm.values[0].jsonPaths=$..metadata.name,$.[?(@.kind == 'Ingress')].spec.rules..http.paths..backend.service.name
```

Comment:
The syntax is very complex. Can we document it please and perhaps add a doc link where developers could discover how to design such JSONpath query ?

Reply:
Added javadoc to ValueReference.jsonPaths with links about how to use it.
For the rest of the document, it will be done in https://github.com/dekorateio/dekorate/issues/842 once the Dekorate site is merged.